src/DataService/Annotation/DataService.php

```diff
@@ -97,4 +104,14 @@ public function hasBaseUri(): bool
     {
         return (bool) $this->baseUri;
     }
+
+    /**
+     * Return if this service is only available over HTTP.
+     *
+     * @return bool
+     */
+    public function isInsecure(): bool
```

Comment:
We already have $insecure as a public property, why do you implement this?

Reply:
Total and complete habit from creating protected / private properties. Will remove!